container/lxd/initialisation_linux.go

```diff
@@ -79,21 +77,36 @@ func (ci *containerInitialiser) Initialise() error {
 	if err := ensureDependencies(ci.lxdSnapChannel, localSeries); err != nil {
 		return errors.Trace(err)
 	}
-	err = ci.configureLxdBridge()
-	if err != nil {
-		return errors.Trace(err)
-	}
 	proxies := proxy.DetectProxies()
 	err = ci.configureLxdProxies(proxies, ci.isRunningLocally, ci.newLocalServer)
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	output, err := ci.getExecCommand(
-		"lxd",
-		"init",
-		"--auto",
-	).CombinedOutput()
+	lxdInitCfg := `config: {}
+networks: []
+storage_pools:
+- config:
+    size: 5GiB
+  description: ""
+  name: default
+  driver: zfs
```

Comment:
Is this safe to do? We'd need to check what lxd does in this scenario... i.e. what happens if zfs isn't available, but btrfs is (this might not be an issue for > 5.x)

Reply:
You are totally right, it was a mistake. I updated it to `dir`, because according to LXD folks `init --auto` always creates a `dir` storage anyway. Thanks for spotting!